config.wyam

```diff
@@ -148,6 +148,7 @@ Pipelines.Add("Blogs",
         Take(5)),
     Yaml(),    
     Content(string.Empty),
+    Execute((inputs, ctx, _) => GetOpmlData(inputs, ctx)),

```

Comment:
I think we can simplify this even further - you may not even need `GetOpmlData()` now that you're relying on metadata from the "Blogs" pipeline in the `feeds.cshtml` file. It looks like the `GetOpmlData()` method is mostly responsible for setting a "Title" metadata value when one doesn't exist. That's actually on of the jobs of the `GetFeedData()` method right below it - and that method sets the "Title" from the actual RSS feed. In fact, the current ordering will actually prevent `GetFeedData()` from setting the title from the RSS because it only does so if an explicit title isn't set and `GetOpmlData()` right before it sets the "Title" metadata in that case (so `GetFeedData()` would never get a blank title that signals the RSS one should be used). At a minimum, `GetOpmlData()` should be moved below `GetFeedData()` so it only sets the backup title if one wasn't provided in the YAML _and_ couldn't be obtained from the RSS feed.